src/components/common/Layout.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router-dom';
+
+import Header from './Header';
```

Comment:
여기도 절대경로로 맞춰주시면 좋을 것 같습니다
```suggestion
import Header from 'components/common/Header';
```